python/moz/l10n/paths/config.py

```diff
@@ -207,6 +207,24 @@ def locales(self, locales: list[str] | None) -> None:
         for incl in self._includes:
             incl.locales = locales
 
+    @property
+    def all_locales(self) -> set[str]:
+        """
+        All locales for the config,
+        including locales set in its imports and paths.
```

Comment:
Why `and paths`?

Reply:
Because that's what compare-locales also does:
https://github.com/mozilla/compare-locales/blob/41e148f6ed6416f65f0e4592169b4745853f52cf/compare_locales/paths/project.py#L146-L148